src/features/people/ActorTile/styled.js

```diff
@@ -41,10 +41,12 @@ export const ActorName = styled.h3`
   font-weight: 500;
   margin: 12px 0 0 0;
   color: ${({ theme }) => theme.color.woodsmoke};
+  height: 64px;
 
   @media (max-width: ${bpMobile}px) {
     font-size: 14px;
     margin: 8px 0 0 0;
+    height: 36px;
```

Comment:
I wonder whether we should give fixed heights. Try to give `StyledActorTile` `height: 100%` and that should fix the problem